src/communication/network_interface_cl.c

```diff
@@ -10561,13 +10561,62 @@ loaddb_update_stats ()
 #endif /* !CS_MODE */
 }
 
+static int
+unpacking_summary_entry (char **ptr, Map_Summary & summary)
+{
+  TRANID trid;
+  char *user = NULL;
+  time_t start_time, end_time;
+  int num_insert, num_update, num_delete;
+  LOG_LSA start_lsa, end_lsa;
+  int num_table;
+  OID tablelist[32];
+
+  char *tmp_ptr = *ptr;
+
+  FLASHBACK_SUMMARY_INFO *info = (FLASHBACK_SUMMARY_INFO *) malloc (sizeof (FLASHBACK_SUMMARY_INFO));
```

Comment:
yes, it seems nice to use `unique_ptr`. 
I have to implement constructor for `flashback_summary_info` first.

For now, this is just test code for summary information is returned well from server, 
so I will deal `unique_ptr` when printing the summary information. I need some more time to use it because I haven't used it before.

Reply:
Which part is a test code? I don't think we need to review the test code.

Reply:
this function (unpacking_summary_entry) is test code, and will be used similarly in other issue (print flashback summary information). I'm sorry to not mention this.

Reply:
ok